### PR TITLE
#167730731 Travis CI Configuration

### PR DESCRIPTION
**Pull request by OlawaleJoseph**
#### What does this PR do?
  This PR Integrates Travis CI into the project
#### Description of Task to be completed?
  - Integrate Travis into the repo
  - Activate the repo on [Travis website](https://travis-ci.com)
  - Create .travis.yml file in the root of the project

#### How should this be manually tested?
  - git clone this repo using the command `git clone git@github.com:andela/firestar-backend.git`
  - cd to the project
  - checkout to the branch using the command `git checkout ch-travis-setup-167730731`
  - Make changes to the branch, git add the changes, commit the changes and push to origin, to trigger build on travis.
  - Open your browser and visit [Travis](https://travis-ci.com/andela/firestar-backend)
  - Give it a moment to build and after a few seconds the build will pass
Kindly view below for more info
#### Any background context you want to provide?
The branch on Travis CI badge in the ReadMe file refers to the stage branch which currently has no test, hence, the build failing badge been displayed. After merging this branch to stage, that will be corrected.
Screenshot (7)

#### What are the relevant pivotal tracker stories?
[#167730731](https://www.pivotaltracker.com/story/show/167730731)

#### Screenshots (if appropriate)

Screenshot (4)

#### Questions:
N/A

### CODE REVIEW

**Review by dewaleolaoye — CHANGES_REQUESTED** [review]
Please, kindly write the test
the reviewer only check the functionality of the feature then give feedback

File: test/travis.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const testFunc = require('../src/travis_test');
+const chai = require('chai');
+const assert = chai.assert;
+describe('Travis ', () => {
+    it('Should be equal', () => {
+        assert.equal(testFunc(200), 200, 'The values should be equal')
+    })
+
+})
```
[the remark this answers is not in the document]
Comment (OlawaleJoseph) [reply test/travis.js:9]: This is a dummy file to be deleted

File: test/travis.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const testFunc = require('../src/travis_test');
+const chai = require('chai');
+const assert = chai.assert;
+describe('Travis ', () => {
+    it('Should be equal', () => {
+        assert.equal(testFunc(200), 200, 'The values should be equal')
+    })
```
[the remark this answers is not in the document]
Comment (OlawaleJoseph) [reply test/travis.js:7]: This is a dummy file to be deleted

File: test/travis.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+const testFunc = require('../src/travis_test');
+const chai = require('chai');
+const assert = chai.assert;
+describe('Travis ', () => {
+    it('Should be equal', () => {
+        assert.equal(testFunc(200), 200, 'The values should be equal')
```
[the remark this answers is not in the document]
Comment (OlawaleJoseph) [reply test/travis.js:6]: This is a dummy file to be deleted

File: test/travis.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+const testFunc = require('../src/travis_test');
+const chai = require('chai');
+const assert = chai.assert;
+describe('Travis ', () => {
+    it('Should be equal', () => {
```
[the remark this answers is not in the document]
Comment (OlawaleJoseph) [reply test/travis.js:5]: This is a dummy file to be deleted

File: test/travis.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+const testFunc = require('../src/travis_test');
+const chai = require('chai');
+const assert = chai.assert;
```
[the remark this answers is not in the document]
Comment (OlawaleJoseph) [reply test/travis.js:3]: This is a dummy file to be deleted

File: test/travis.js [hunk 6]
```diff
@@ -0,0 +1,9 @@
+const testFunc = require('../src/travis_test');
+const chai = require('chai');
```
[the remark this answers is not in the document]
Comment (OlawaleJoseph) [reply test/travis.js:2]: This is a dummy file to be deleted

File: test/travis.js [hunk 7]
```diff
@@ -0,0 +1,9 @@
+const testFunc = require('../src/travis_test');
```
[the remark this answers is not in the document]
Comment (OlawaleJoseph) [reply test/travis.js:1]: This is a dummy file to be deleted

File: src/travis_test.js [hunk 8]
```diff
@@ -0,0 +1,3 @@
+const sayhello = (str) => str;
```
[the remark this answers is not in the document]
Comment (OlawaleJoseph) [reply src/travis_test.js:1]: This is a dummy file to be deleted

**Comment by Dolapo47** [comment]
Good work

**Review by Dolapo47 — APPROVED** [review]
Good work

**Review by dewaleolaoye — APPROVED** [review]
Well done!

**Pull request closed without merging**